packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -19,12 +19,12 @@ import 'raw_keyboard.dart';
 class RawKeyEventDataLinux extends RawKeyEventData {
   /// Creates a key event data structure specific for macOS.
   ///
-  /// The [toolkit], [scanCode], [codePoint], [keyCode], and [modifiers], arguments
-  /// must not be null.
+  /// The [toolkit], [scanCode], [codePoint], [keyCode], and
+  /// [modifiers], arguments must not be null.
```

Comment:
maybe you have your line width set very short? this seems like a surprising change also

Reply:
Ditto from comment above